packages/integration-boilerplate-react/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export {
+  default as usePenpal,
+  PenpalContextProvider,
+  ConfigMode,
```

Comment:
It might be worth adding `PenpalContext` and `PenpalContextValue` as exports, the salesforce frontend tests needed them but I just imported from the dist folder since it was only for tests.